### PR TITLE
Add CHANGELOG message for PR #41283

### PR DESCRIPTION
**Pull request by tywhang**
### Summary

Add CHANGELOG entry for #41283

### CODE REVIEW

File: actionview/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+*   Improve error messages when file does not exist at filepath.
```
Comment (p8) [inline actionview/CHANGELOG.md:1]:
```suggestion
*   Improve error messages when template file does not exist at absolute filepath.
```

Comment (tywhang) [reply actionview/CHANGELOG.md]: Love the suggestion!

Comment (p8) [reply actionview/CHANGELOG.md]: Thanks 😄

Comment (p8) [reply actionview/CHANGELOG.md]: and thanks for the PR! 🙇

**Review by zzak — APPROVED** [review]
Thanks @tywhang! :bow:

**Pull request merged**